org.adempiere.base/src/org/adempiere/base/sso/SSOUtils.java

```diff
@@ -0,0 +1,64 @@
+/******************************************************************************
+ * Copyright (C) 2016 Logilite Technologies LLP								  *
+ * This program is free software; you can redistribute it and/or modify it    *
+ * under the terms version 2 of the GNU General Public License as published   *
+ * by the Free Software Foundation. This program is distributed in the hope   *
+ * that it will be useful, but WITHOUT ANY WARRANTY; without even the implied *
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.           *
+ * See the GNU General Public License for more details.                       *
+ * You should have received a copy of the GNU General Public License along    *
+ * with this program; if not, write to the Free Software Foundation, Inc.,    *
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA.                     *
+ *****************************************************************************/
+
+package org.adempiere.base.sso;
+
+import java.util.List;
+
+import org.adempiere.base.Service;
+import org.compiere.model.I_SSO_PrincipleConfig;
+import org.compiere.model.MSSOPrincipleConfig;
+
+/**
+ * @author Logilite Technologies
+ */
+public class SSOUtils
+{
+	public static final String	ERROR_API				= "/error.html";
+
+	public static final String	ERROR_VALIDATION		= "/error.zul";
```

Comment:
probably better to named as ERROR_URL and ERROR_VALIDATION_URL instead.

Reply:
Done